src/ui/components/modal/ExportDataModal.tsx

```diff
@@ -65,36 +68,37 @@ export const ExportDataModal: FC<ExportDataModalProps> = ({
           <SettingsEditModal
             onClose={onClose}
             isOpen={isOpen}
-            title={variant === 'export' ? 'Export data' : 'Email data'}
+            title={`${t('CREATING_REPORT_TITLE')}:\n ${formatter.dateRange(from, to)}`}
             onConfirm={wrappedOnConfirm}
             isDisabled={!isValid}
             Content={
               <Column alignItems="center" flex={1} justifyContent="center">
                 <Input
                   ref={usernameRef}
                   value={values.username}
-                  label="Username"
-                  placeholder="Enter your name"
+                  label={t('USERNAME')}
+                  placeholder={t('ENTER_YOUR_USERNAME')}
                   onChangeText={e => {
                     handleChange('username')(e);
                   }}
-                  inputContainerStyle={{ width: width * 0.5 }}
+                  inputContainerStyle={{ width: width * 0.8 }}
```

Comment:
could the magic number be moved to the constants?

Reply:
thanks. 
cdf8025 

I don't think I like this way of styling hehe